src/Sarif.Converters/StaticDriverVerifierConverter.cs

```diff
@@ -228,14 +265,26 @@ private void ProcessLine(string logFileLine, Result result)
             }
         }
 
-        private string BuildMessageFromState(string state)
+        private static Regex s_callRegEx = new Regex(@"Call '(.*)' '(.*)'", RegexOptions.Compiled);
+        private bool IsImportantCall(string fileName, string extraMsg)
         {
-            if (string.IsNullOrEmpty(state) || state == "^")
+            Match match = s_callRegEx.Match(extraMsg);
+            if (match.Success && match.Groups.Count == 3)
             {
-                return null;
+                return (!IsHarnessOrRuleFile(fileName) && !match.Groups[1].Value.StartsWith("SLIC_"));
```

Comment:
You could test `IsHarnessOrRuleFile` at the top of the method and save the effort of evaluating the regex if the file was unimportant.